app/src/main/java/comcmput301f16t01/github/carrier/driverViewRequestActivity.java

```diff
@@ -3,7 +3,7 @@
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
 
-public class driver_view_request extends AppCompatActivity {
+public class DriverViewRequestActivity extends AppCompatActivity {
```

Comment:
Idk haha. I got an error from there on build and just commented it out to run and it worked. I could be crazy.